tests/tools/oesign/test-engine/CMakeLists.txt

```diff
@@ -7,12 +7,20 @@ target_include_directories(oesign_test_engine PRIVATE include)
 
 set(OESIGN_TEST_INPUTS_DIR ${CMAKE_CURRENT_BINARY_DIR}/../test-inputs)
 
-configure_file(../sign-and-verify.py sign-and-verify.py COPYONLY)
+add_custom_command(
+  OUTPUT sign-and-verify.py
+  COMMAND cmake -E copy ${CMAKE_CURRENT_SOURCE_DIR}/../sign-and-verify.py
```

Comment:
`DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/../sign-and-verify.py`

Reply:
Added